plugins/inputs/x509_cert/x509_cert.go

```diff
@@ -187,25 +192,43 @@ func getTags(cert *x509.Certificate, location string) map[string]string {
 	return tags
 }
 
+// copied from plugins/inputs/file/file.go
+func (c *X509Cert) refreshFilePaths() error {
+	var allFiles []string
+
+	for _, source := range c.Sources {
+		if strings.HasPrefix(source, "/") {
```

Comment:
I think you should swap the order of processing and _first_ call `sourcesToURLs()` and afterwards this function. This way you can be sure that _file_ always have `file://` prepended. Otherwise you might miss files where the user added `file://` already for you.
```suggestion
		if strings.HasPrefix(source, "file://") {
```

Reply:
the globing tools don't work with url styled path, so I'll have to strip the leading file://